### ISSUE TITLE
Make quickstart examples for R

### ISSUE THREAD

**Issue by mccalluc** [opendp/opendp#1232]
There's a bigger documentation project in draft:
- #1177

but we wanted to wait on that until polars is in.

Before then, it would be good to demonstrate how sphinx tabs will work, and apply it to the quickstart page.

- first: get the tabs up, with R examples
- second, figure out something like doctests for R, so we're sure the code works (might be split to a separate issue)

**Comment by Shoeboxam** [comment]
I think the solution might involve adding directives? 
https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/doctest.py

You can access `app` to add a directive in `def setup(app)` within the `conf.py`.
